Lua/LuaAdapters.cpp

```diff
@@ -307,6 +307,46 @@ namespace RTE {
 		luaSelfObject->GibThis();
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	const std::vector<AEmitter *> & LuaAdaptersMOSRotating::GetWounds1(const MOSRotating *luaSelfObject) {
+		return GetWounds2(luaSelfObject, true, false, false);
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	const std::vector<AEmitter *> & LuaAdaptersMOSRotating::GetWounds2(const MOSRotating *luaSelfObject, 
+	                                                                  bool includePositiveDamageAttachables,
+	                                                                  bool includeNegativeDamageAttachables, 
+	                                                                  bool includeNoDamageAttachables) {
+		static std::vector<AEmitter *> wounds;
+		wounds.clear();
```

Comment:
I'd much prefer it being in the function itself. It's self-contained and it doesn't touch anything. It's being absolutely clear that this is something directly related to this function specifically. You can delete this function in isolation without leaving a mess that you forget about.

I'm not sure what's wrong with the practise - I've seen it done very often in many codebases, primarily as an optimization technique to avoid memory allocations, so it's pretty obvious to me. Some codebases use a macro for it, LOCAL_STATIC_CONTAINER or something, that handles clearing it and all that stuff automatically, but I don't think we do that enough to justify it yet.

Reply:
As we discussed, put in an explanatory comment for those less familiar with this sort of setup.